doc/api/worker_threads.md

```diff
@@ -252,6 +252,111 @@ if (isMainThread) {
 }
 ```
 
+## `worker.postMessageToWorker(destination, value[, transferList][, timeout])`
```

Comment:
The name is a bit misleading as this API allows also to send a message to main thread not just to other workers.

Reply:
I kinda agree. But I can't find a better name. Any idea?

Reply:
What about `postMessageToThread` and rename parameter `destination` to `threadId`?

Reply:
I'm fine with it. @nodejs/tsc Are we in agreement on the name?

Reply:
This is not a question for the TSC

Reply:
bikeshedding wise - postMessageToThread sounds better to ne too

Reply:
@targos I meant to tag several people, all of them in TSC, so I went out of laziness. My mistake :)

Reply:
Ok, I've renamed it.